e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/OrderTest.groovy

```diff
@@ -40,6 +40,27 @@ class OrderTest extends Specification {
         parsedSentPayload.entry[24].resource.contact.name.text.contains("SADIE S SMITH")
     }
 
+    def "check that ETOR processing code is added to the order before sending to report stream"() {
```

Comment:
Do we need to add an e2e test for the results flow as well?

Reply:
This PR only covers the metadata tag. I added e2e for the Orders use case because the code allowed for it. We still need the code to send to RS and then we can add the e2e for Results.